cupyx/scipy/special/convex_analysis.py

```diff
@@ -0,0 +1,129 @@
+from cupy import core
+
+
+_float_preamble = '''
+#ifndef NAN
+#define NAN __int_as_float(0x7fffffff)
+#endif
+
+#ifndef INF
+#define INF __int_as_float(0x7f800000)
+#endif
+
+
+double __device__ entr(double x) {
+    if(isnan(x)) {
```

Comment:
Nitpick
```suggestion
    if (isnan(x)) {
```
Please fix other places as well.